test/src/test/java/org/corfudb/integration/StreamIT.java

```diff
@@ -15,54 +16,33 @@
  */
 
 public class StreamIT extends AbstractIT {
-    static String corfuSingleNodeHost;
-    static int corfuSingleNodePort;
 
-    @Before
-    public void loadProperties() throws Exception {
-        corfuSingleNodeHost = (String) PROPERTIES.get("corfuSingleNodeHost");
-        corfuSingleNodePort = Integer.parseInt((String) PROPERTIES.get("corfuSingleNodePort"));
-    }
-
-//    @Test
-    public void simpleStreamTest() throws Exception {
-
-        Process corfuServerProcess = new CorfuServerRunner()
-                .setHost(corfuSingleNodeHost)
-                .setPort(corfuSingleNodePort)
+    @Test
+    public void largeStreamWrite() throws Exception {
+        final String host = "localhost";
+        final String streamName = "s1";
+        final int port = 9000;
+
+        // Start node one and populate it with data
+        Process server_1 = new CorfuServerRunner()
+                .setHost(host)
+                .setPort(port)
+                .setSingle(true)
                 .runServer();
 
-        CorfuRuntime rt = createDefaultRuntime();
-        rt.setCacheDisabled(true);
-
-        Random rand = new Random();
-
-        UUID streamId = CorfuRuntime.getStreamID(Integer.toString(rand.nextInt()));
-
-        IStreamView s1 = rt.getStreamsView().get(streamId);
-
-        // Verify that the stream is empty
-        assertThat(s1.hasNext())
-                .isFalse();
-
-        // Generate and append random data
-        int entrySize = Integer.valueOf(PROPERTIES.getProperty("largeEntrySize"));
-        final int numEntries = 100;
-        byte[][] data = new byte[numEntries][entrySize];
-
-        for(int x = 0; x < numEntries; x++) {
-            rand.nextBytes(data[x]);
-            s1.append(data[x]);
-        }
+        final int maxWriteSize = 100;
 
-        // Read back the data and verify it is correct
-        for(int x = 0; x < numEntries; x++) {
-            ILogData entry = s1.nextUpTo(x);
-            byte[] tmp = (byte[]) entry.getPayload(rt);
+        CorfuRuntime.CorfuRuntimeParameters params = CorfuRuntime.CorfuRuntimeParameters
+                .builder()
+                .maxWriteSize(maxWriteSize)
+                .build();
 
-            assertThat(tmp).isEqualTo(data[x]);
-        }
+        CorfuRuntime rt = CorfuRuntime.fromParameters(params);
+        rt.parseConfigurationString(host + ":" + port);
+        rt.connect();
 
-        assertThat(shutdownCorfuServer(corfuServerProcess)).isTrue();
+        final int bufSize = maxWriteSize * 2;
+        rt.getStreamsView().get(CorfuRuntime.getStreamID(streamName)).append(new byte[bufSize]);
```

Comment:
I do not see any checks. Am I missing something.

Reply:
pr wasnt ready. Now it should be.